src/plays/model/Plays.ts

```diff
@@ -0,0 +1,6 @@
+export default interface Plays {
```

Comment:
There is no need to add `Plays` as a separate model as `Play` have `steps` as an `array` which can be empty as well.
This way you can use play methods as well.